status-im/status-react
# feat: implement onboarding modal transition for sign in screen

fixes #16101

### Summary

Implements modal transition for sign in screen in the onboarding

#### Results


#### Platforms

- Android
- iOS

### Steps to test

- Open Status (fresh install)
- Tap on "Sign In"
- Verify transition / animation

status: ready
